tsconfig.json

```diff
@@ -3,10 +3,10 @@
       "paths": { "matrix-js-sdk": ["node_modules/@types/matrix-js-sdk/index.d.ts"] },
       /* Basic Options */
       // "incremental": true,                   /* Enable incremental compilation */
-      "target": "es2017",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+      "target": "es2020",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
It's better to tell which Node.js version you're targetting with this in the comments

https://github.com/paritytech/try-runtime-bot/blob/28cf9a41157543fe0fe0a6dcbc0b147f95f4aab8/tsconfig.json#L11